docker-compose.yml

```diff
@@ -0,0 +1,97 @@
+version: "3.8"
+
+services:
+  khan:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+      elasticsearch:
+        condition: service_healthy
+    links:
+      - postgres
+      - redis
+      - elasticsearch
+    ports:
+      - "80:80"
+    networks:
+      - khan
+    environment:
+      - KHAN_POSTGRES_HOST=postgres
+      - KHAN_POSTGRES_USER=postgres
+      - KHAN_POSTGRES_PASSWORD=123456
+      - KHAN_ELASTICSEARCH_HOST=elasticsearch
+
+  migrate:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: "make migrate"
```

Comment:
Using this configuration with a `docker-compose up` without reading README, `migrate` and `khan` will be running concurrently. Could we have problems with the `khan` container due to this?
Which is the advantage of running `migrate` outside the `khan` container?

Reply:
Good point. I can override the command to execute both make commands instead of using two separated services.